hackathon_site/dashboard/frontend/src/components/teamDetail/TeamInfoTable/TeamInfoTable.tsx

```diff
@@ -0,0 +1,107 @@
+import { RouteComponentProps } from "react-router-dom";
+
+import {
+    Checkbox,
+    Container,
+    Grid,
+    Paper,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+} from "@material-ui/core";
+import Typography from "@material-ui/core/Typography";
+import { mockTeamMultiple } from "testing/mockData";
+import React from "react";
+import { PageParams } from "pages/TeamDetail/TeamDetail";
+
+export const TeamInfoTable = ({ match }: RouteComponentProps<PageParams>) => {
```

Comment:
I think I may not have been clear, this component should only hold code for the **team info table**, not any other tables or headings. Some of this code (like the Team Overview title) is more associated with the page than the table.

Basically, everything from line 30 to 81 is the actual Team Info Table and the rest of the code should go back into TeamDetail.tsx

Reply:
fixed